app/src/main/java/org/gdg/frisbee/android/chapter/NewsFragment.java

```diff
@@ -83,7 +85,7 @@ public void onActivityCreated(Bundle savedInstanceState) {
         if (Utils.isOnline(getActivity())) {
             setIsLoading(true);
             App.getInstance().getModelCache()
-                .getAsync(Const.CACHE_KEY_NEWS + plusId, new ModelCache.CacheListener() {
+                .getAsync(CACHE_KEY_NEWS + plusId, new ModelCache.CacheListener() {
```

Comment:
I don't think we can enforce this properly. 

Shall we think of the keys as part of the cache or as part of the user of the cache? Difficult question. I tend towards the cache. So I'd prefer to go towards a cache that has knowledge of the model instead of cache users that have knowledge of all other cache users (at least of the convention that was agreed between them)